test/sai_test/sai_fdb_test.py

```diff
@@ -65,3 +67,1393 @@ def tearDown(self):
+        print("\tVerification complete")
+
+    def tearDown(self):
+        sai_thrift_remove_fdb_entry(self.client, self.fdb_entry)
+        sai_thrift_remove_fdb_entry(self.client, self.moving_fdb_entry)
+        status = sai_thrift_flush_fdb_entries(self.client, entry_type=SAI_FDB_FLUSH_ENTRY_TYPE_DYNAMIC)
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
+
+
+class FdbStaticMacMoveTest(T0TestBase):
+    '''
+    Verify if disable MAC move, drop packet with known
+    '''
+
+    def setUp(self):
+        """
+        Test the basic setup process
+        """
+        T0TestBase.setUp(self, is_reset_default_vlan=False)
+        sai_thrift_flush_fdb_entries(
+            self.client,
+            entry_type=SAI_FDB_FLUSH_ENTRY_TYPE_ALL)
+        self.fdb_entry1 = sai_thrift_fdb_entry_t(switch_id=self.switch_id,
+            mac_address=self.local_server_mac_list[2],
+            bv_id=self.vlans[10].vlan_oid)
+        status = sai_thrift_create_fdb_entry(self.client,
+            self.fdb_entry1,
+            type=SAI_FDB_ENTRY_TYPE_STATIC,
+            bridge_port_id=self.bridge_port_list[2])
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
+
+        self.fdb_entry2 = sai_thrift_fdb_entry_t(switch_id=self.switch_id,
+            mac_address=self.local_server_mac_list[1],
+            bv_id=self.vlans[10].vlan_oid)
+        status = sai_thrift_create_fdb_entry(self.client,
+            self.fdb_entry2,
+            type=SAI_FDB_ENTRY_TYPE_DYNAMIC,
+            bridge_port_id=self.bridge_port_list[1],
+            allow_mac_move=True)
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
+
+        self.pkt = simple_udp_packet(eth_dst=self.local_server_mac_list[2],
+            eth_src=self.local_server_mac_list[1],
+            pktlen=100)
+
+    def runTest(self):
+        send_packet(self, 1, self.pkt)
+        verify_packet(self, self.pkt, self.dev_port_list[2])
+
+        # inititally add moving MAC to FDB
+        self.moving_fdb_entry = sai_thrift_fdb_entry_t(
+            switch_id=self.switch_id,
+            mac_address=self.local_server_mac_list[1])
+        status = sai_thrift_create_fdb_entry(
+            self.client,
+            self.moving_fdb_entry,
+            type=SAI_FDB_ENTRY_TYPE_STATIC,
+            bridge_port_id=self.bridge_port_list[3],
+            allow_mac_move=True)
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
+        send_packet(self, 3, self.pkt)
+        verify_packet(self, self.pkt, self.dev_port_list[2])
+
+        print("\tVerification complete")
+
+    def tearDown(self):
+        sai_thrift_remove_fdb_entry(self.client, self.fdb_entry1)
+        sai_thrift_remove_fdb_entry(self.client, self.fdb_entry2)
+        sai_thrift_remove_fdb_entry(self.client, self.moving_fdb_entry)
+        status = sai_thrift_flush_fdb_entries(self.client, entry_type=SAI_FDB_FLUSH_ENTRY_TYPE_DYNAMIC)
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
```

Comment:
no blank line

Reply:
those codes with the same purpose and within same logic, then put then in same block (without break lines)
https://peps.python.org/pep-0008/#blank-lines